src/Config.zig

```diff
@@ -328,6 +328,8 @@ test Field {
     try walkFields(@This(), "");
 }
 
+/// Provides metadata for the field, including its type, description,
```

Comment:
```suggestion
/// Provide field type of config fields and its metadata.
```

The only important description is the `Drivercom.Config.Info` is actually just a metadata for config fields. What kind of metadata given is not important to be written for `Drivercom.Config.Info` doc comments as it is clearly written in `Drivercom.Config.Info.Meta`. Maybe we can some information as well that the metadata of a field is given with prefix double underscore.